docs/usage/usage.md

```diff
@@ -229,7 +229,7 @@ In contrast to a preprocessing function, a tool usually adds an easily interpret
 
 ## Plotting
 
-The plotting module ehrapy.p**l largely parallels the tl.\* and a few of the pp.\* functions.
+The plotting module ehrapy.p\*\*l largely parallels the tl.\* and a few of the pp.\* functions.
```

Comment:
Huh, where is this change coming from?

Reply:
Tbh, idk. Just ran nox and somehow this appeared.

Reply:
Yeah, I figured this. We should examine whether this still renders properly

Reply:
renders incorrect, will update this